DMTN-276.tex

```diff
@@ -53,88 +56,263 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
-\section{Camera Level Persistence}
-Persistence was first found in Run 5 of electro-optical (EO) testing at SLAC. Further investigation found that this effect only affected E2V sensors. 
-This persistence was found to have an average signal after the flash of 6 ADU and had a decay constant of 37 seconds. 
-It was also foud that the persistence only went away with integration time. One of the unique features of this persistence is the persistence trail. 
+\section{Electro-optical Measured Persistence}
+% \begin{figure*}[!htp]
+%   \centering
+%   \includegraphics[width=0.95\textwidth, angle=0]{Run_5_persistence_ex.png}
+%   \caption{
+%   Example of peristence in Run 5 data. 
+%   This figure shows the initial flash on the left most image (dataId shown as the title) and the subsquent darks images going from left to right.
+%   This particular flash has one of the three spots inbetween two amplifiers and you can see the trail starting at the spots and going all the way to the readout of each amplifier.
+%   }\label{fig:ex_persistence_Run5}
+% \end{figure*}
+
+\subsection{Definition and Run 5 Measurements}
+
+\begin{figure*}[!htp]
+  \centering
+  \includegraphics[width=0.95\textwidth, angle=0]{Run_5_persistence_ex.png}
+  \caption{
+  Example of peristence in Run 5 data. 
+  This figure shows the initial flash on the left most image (dataId shown as the title) and the subsquent darks images going from left to right.
+  This particular flash has one of the three spots inbetween two amplifiers and you can see the trail starting at the spots and going all the way to the readout of each amplifier.
+  }\label{fig:ex_persistence_Run5}
+\end{figure*}
+
+
+
+Persistence is a sensor level effect where charge gets trapped from one image and `persists' into the next at some fraction of the original charge. 
+Figure \ref{fig:ex_persistence_Run5} shows an example of this from Run 5 data where three saturated spots persist into subsequent dark images. 
+Persistence for LSSTCam was first found in Run 5 of electro-optical (EO) testing at SLAC from crosstalk measurements. 
+Further investigation found that this effect only affected E2V sensors
+% This persistence was found to have an average signal after the flash of 6 ADU and had a decay constant of 37 seconds. 
+and that the persistence was only mitigated via integration time as no persistence was shown in biases. 
```

Comment:
So I changed it to try and explain but just in case it is still confusing, I will try and do it here. Essentially, there were some tests done to measure and mitigate the persistence and one of those was taking biases after the bright flash and seeing if that helped with persistence. It didn't as after the biases, there persistence was still visible in a dark image.